app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,19 @@
+.cafe-date {
+  background: $background_color;
```

Comment:
Name of variable `background_color` should be written in all lowercase letters with hyphens instead of underscores